.gitignore

```diff
@@ -1,3 +1,4 @@
 /js/impress.min.js
 /node_modules
 /npm-debug.log
+/.brackets.json
```

Comment:
I use Brackets for JS-Development and impress.js with extras contains very many files (over 30,000). Indexing them all slows down Brackets very much, so I hide the extras directory from Bracktes with the help this file, where I set some exclude dirs. 
Of course we do not need it for impress.js, so I added it to `.gitignore`, to prevent git from pushing it to github.

Reply:
Ok, could you maybe add a comment like this then:

    /js/impress.min.js
    /node_modules
    /npm-debug.log

    +# Files for editors and other tools
    +/.brackets.json